core/src/epicli/data/common/ansible/playbooks/roles/logstash/tasks/main.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: Install logstash-oss {{ specification.version }}
+  include_tasks: "{{ ansible_os_family }}.yml"
+
+- name: Ensure Logstash is disabled
+  systemd:
+    name: logstash
+    enabled: no
```

Comment:
Do I understand correctly that we use it only as a tool executed manually? 🤔

Reply:
Yes. FR was for non scheduled export so for now we only providing **possibility** to export